### ISSUE TITLE
Documentation Suggestion

### ISSUE THREAD

**Issue by JohnOmernik** [d3blocks/d3blocks#17]
I am not sure if it's a mistake in rendering, or something you may not see because you live here, but when I look at the documentation, its really hard to follow.   Example from the link below. 

- param and type actually more like column headers.  But because they are first thing on the line, it's hard to differentiate the variable name from the variable name
- since words like df and color are so generic it gets melded into the words param and type
- Could you use a markdown table that has param, type, example, default, and returns (for fucnctions and then display the variables in a table?

I know this may sound pedantic, but  I really believe your d3blocks is fantastic, and think the way the docs are written may make it harder for folks then it should be to understand what is happening. 

https://d3blocks.github.io/d3blocks/pages/html/Chord.html

```
param df
Input data containing the following columns: ‘source’ ‘target’ ‘weight’ ‘color’ (optional) ‘opacity’ (optional)

type df
pd.DataFrame()

param color
Link colors in Hex notation. Should be the same size as input DataFrame. * ‘source’: Color edges/links similar to that of source-color node. * ‘target’: Color edges/links similar to that of target-color node. * ‘source-target’: Color edges/link based on unique source-target edges using the colormap. * ‘#ffffff’: All links have the same hex color. * [‘#000000’, ‘#ffffff’,…]: Define per link.

type color
(default: ‘source’)

param opacity
Link Opacity. Should be the same size as input DataFrame. * ‘source’: Opacity of edges/links similar to that of source-opacity node. * ‘target’: Opacity of edges/links similar to that of target-opacity node. * 0.8: All links have the same opacity. * [0.1, 0.75,…]: Set opacity per edge/link.```

**Comment by erdogant** [comment]
True. This is a rendering issue from the sphinx pages that I still need to fix.
To keep the documentation within code and external exactly the same (and prevent tons of extra work), I use automodule sphinx functionality.

This is basically what I to generate the documentation pages:

```
.. automodule:: d3blocks.d3blocks.D3Blocks.chord
    :members:
    :undoc-members:

```

**Comment by erdogant** [comment]
I created an update! It was easy, adding tabs :-)

**Comment by erdogant** [comment]
Closing this one. Reopen if I did miss something.
